recipes/gsl/all/conanfile.py

```diff
@@ -0,0 +1,160 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from contextlib import contextmanager
+import os
+
+required_conan_version = ">=1.33.0"
+
+class GslConan(ConanFile):
+    name = "gsl"
+    description = "GNU Scientific Library"
+    homepage = "https://www.gnu.org/software/gsl"
+    topics = ("numerical", "math", "random", "scientific")
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "GPL-3.0-or-later"
+
+    settings = "os", "compiler", "build_type", "arch"
+
+    options = {"shared": [True, False], "fPIC": [True, False]}
+
+    default_options = {"shared": False, "fPIC": True}
```

Comment:
reformatting suggestion.
Trailing space to avoid reformatting tools (such as black) putting all options in one line.
```suggestion
    settings = "os", "compiler", "build_type", "arch"
    options = {
        "shared": [True, False],
        "fPIC": [True, False],
    }
    default_options = {
        "shared": False,
        "fPIC": True,
    }
```

Reply:
gotcha, was wondering why you did that :+1: